gen/teams_provisioning

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+use Perun::Agent;
+use JSON::XS;
+use Data::Dumper;
+
+our $SERVICE_NAME = "teams_provisioning";
+our $PROTOCOL_VERSION = "3.0.0";
+my $SCRIPT_VERSION = "3.0.0";
+
+perunServicesInit::init;
+my $DIRECTORY = perunServicesInit::getDirectory;
+my $data = perunServicesInit::getHashedHierarchicalData;
+
+#Constants
+our $A_RES_ALIAS;             *A_RES_ALIAS =         \'urn:perun:resource:attribute-def:def:o365ResourceAlias';
+our $A_USER_LOGIN;            *A_USER_LOGIN=             \'urn:perun:user:attribute-def:def:login-namespace:mu';
+
+my @teams = ();
+my @resourcesData = $data->getChildElements;
```

Comment:
Whole processing of `$data` has to be updated to work with the new hashed data, since the returned JSON structure is different. See reworked scripts and do the similar like in #590.

Also please update script version afterwards, eg. `3.0.1` -> `3.0.2`.